internal/provider/gce/environ.go

```diff
@@ -232,14 +232,6 @@ func (env *environ) PrepareForBootstrap(ctx environs.BootstrapContext, controlle
 	return nil
 }
 
-// Create implements environs.Environ.
-func (env *environ) Create(ctx envcontext.ProviderCallContext, p environs.CreateParams) error {
```

Comment:
Hi ian, are we not replacing CreateModelResources and adding ValidateProviderForNewModel for gce?

Reply:
Yes, since the exiting Create just returned nil to satisfy the BootStrapEnviron interface and now with this PR  there's no need to have this meaningless boilerplate.